### PR TITLE
Move most pop MPU back in the normal flow

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Chromium has a bug with the balancing algorithm where it can't compute the height of the column container. Which gives:
picture 3

To circumvent the problem, we move the MPU back into normal document flow and centre it horizontally below the headlines:
picture 2

cc @stephanfowler @jbreckmckye

### CODE REVIEW

**Comment by jbreckmckye** [comment]
Do you know if there are any tickets for this issue on the Chromium project? Perhaps they give an indication as to when they can fix this? How did you track down the cause?

**Comment by regiskuckaertz** [comment]
There is none, I'm creating one now. I should create two in fact, one for `column-fill` and one for `calc`. There is a comment on caniuse.com but no source is provided, nevertheless from a few tests it is fairly obvious the new multicol implementation they released a while ago is incomplete.

**Comment by regiskuckaertz** [comment]
Here is what the spec says about `column-fill`:

> In continuous media, this property will only be consulted if the length of columns has been constrained.

`min-height` and `max-height` set constraints on the length of columns, so Chromium should behave like Safari.

File: static/src/javascripts/projects/common/modules/onward/popular.js [hunk 1]
```diff
@@ -41,16 +42,10 @@ define([
         this.fetch(qwery('.js-popular-trails'), 'html');
     };
 
-    MostPopular.prototype.mobileMaximumSlotsReached = function () {
-        return (detect.getBreakpoint() === 'mobile' && $('.ad-slot--inline').length > 1);
-    };
-
     MostPopular.prototype.prerender = function () {
-        if (commercialFeatures.popularContentMPU && !this.mobileMaximumSlotsReached()) {
```
Comment (jbreckmckye) [inline static/src/javascripts/projects/common/modules/onward/popular.js]: We no longer care about the ads-per-page count on mobile?

Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/onward/popular.js]: A while ago I removed the limitation, but now that you mention it, that was for inline MPUs in article, not fronts. I'm putting this back. Thanks!

File: static/src/stylesheets/module/facia/_slices.scss [hunk 2]
```diff
@@ -239,6 +210,29 @@ Hence why a greater depth of selector specificity is needed.
     }
 }
 
+.fc-slice__popular-mpu {
+    .ad-slot {
+        min-height: 274px;
```
Comment (jbreckmckye) [inline static/src/stylesheets/module/facia/_slices.scss]: Where does 274px come from? I notice _items.scss has the following entry -

    .fc-slice__item--mpu-candidate {
        @include mq(tablet) {
            min-height: 25p + $mpu-ad-label-height;
        }
    }

I don't know if perhaps something could be rationalized between them...

Comment (regiskuckaertz) [inline static/src/stylesheets/module/facia/_slices.scss]: That's a good point. Same for the `286px` a few lines below. I'll try and see what we can do here.

Comment (jbreckmckye) [inline static/src/stylesheets/module/facia/_slices.scss]: Well, it's all existing code, so it's okay if we don't have the chance to resolve it in this PR - but any cleanup is always an improvement!

**Comment by regiskuckaertz** [comment]
@jbreckmckye turns out there are vars for a standard MPU size. We should perhaps think about creating a map with all our sizes—both standard and Guardian-specific.

**Comment by jbreckmckye** [comment]
:+1:

**Pull request merged**